use-timescale/extensions/index.md

```diff
@@ -13,50 +13,120 @@ supported extensions:
-| btree_gist                   |Support for indexing common datatypes in GiST|
-| citext                       |Data type for case-insensitive character strings|
-| cube                         |Data type for multidimensional cubes|
-| dict_int                     |Text search dictionary template for integers|
-| dict_xsyn                    |Text search dictionary template for extended synonym processing|
-| fuzzystrmatch                |Determine similarities and distance between strings|
-| hstore                       |Data type for storing sets of (key, value) pairs|
-| intarray                     |Functions, operators, and index support for 1-D arrays of integers|
-| isn                          |Data types for international product numbering standards|
-| lo                           |Large Object maintenance|
-| ltree                        |Data type for hierarchical tree-like structures|
-| pg_stat_statements           |Track execution statistics of all SQL statements executed|
-| pg_trgm                      |Text similarity measurement and index searching based on trigrams|
-| pgaudit                      |Detailed session and/or object audit logging|
-| [pgcrypto][pgcrypto]         |Cryptographic functions|
-| pgpcre                       |Perl-compatible RegEx|
-| pgrouting                    |pgRouting Extension|
-| pgstattuple                  |Obtain tuple-level statistics|
-| [pgvector][pgvector]         |Vector similarity search for PostgreSQL|
-| plpgsql                      |SQL procedural language|
-| [postgis][postgis]           |PostGIS geometry and geography spatial types and functions|
-| postgis_raster               |PostGIS raster types and functions|
-| postgis_sfcgal               |PostGIS SFCGAL functions|
-| postgis_tiger_geocoder       |PostGIS tiger geocoder and reverse geocoder|
-| postgis_topology             |PostGIS topology spatial types and functions|
-| [postgres_fdw][postgres-fdw] |Foreign data wrappers|
-| seg                          |data type for representing line segments or floating-point intervals|
-| tablefunc                    |Functions that manipulate whole tables, including crosstab|
-| tcn                          |Triggered change notifications|
-| timescaledb_toolkit          |TimescaleDB Toolkit|
-| timescaledb                  |Enables scalable inserts and complex queries for time-series data|
-| tsm_system_rows              |TABLESAMPLE method which accepts number of rows as a limit|
-| tsm_system_time              |TABLESAMPLE method which accepts time in milliseconds as a limit|
-| unaccent                     |Text search dictionary that removes accents|
-| unit                         |SI Units for PostgreSQL|
-| uuid-ossp                    |Generate universally unique identifiers (UUIDs)|
+| Extension                                        | Description                                                            | Enabled for service capability |
```

Comment:
Can we rephrase "enabled for service capability" to something more user-friendly, i.e. preinstalled?

Reply:
@billy-the-fish ?

Reply:
Hi @alexeyklyukin, happy to change the words, but we do need something to express what is happening. The fact that an extension is in the list means that it is preinstalled. Perhaps if we change this to say simply Enabled?

Reply:
@billy-the-fish thanks, indeed, "Precreated" would likely be a better term, or perhaps "Enabled by default"?

Reply:
Enabled by default